app/index.ejs

```diff
@@ -29,7 +29,7 @@ body { margin: 0; }
   justify-content: center;
   width: 100vw;
   height: 100vh;
-  background: #000;
+  background: hsl(233, 36%, 13%);
```

Comment:
Would it be possible to make these colors relative to our stylesheet? Maybe build a css-file and then render it in at build-time of the index.html? (not high prio)

Reply:
Yes, we can do that later.